v2/blockstore/readonly_test.go

```diff
@@ -18,13 +18,15 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+var bg = context.Background()
```

Comment:
please don't use a global context; I know it's a no-op and just used in tests, but I would rather avoid sharing any form of state between tests.